packages/cli/src/github/github.ts

```diff
@@ -156,21 +154,29 @@ export class Github {
    * Get all Github releases and match their tag names against the given semver
    * range. Return the release with the latest possible match.
    */
-  async getSemverRelease(semverRange: string): Promise<GitHubApi.Release> {
+  async getSemverRelease(semverRange: string): Promise<any> {
```

Comment:
Switching from types to `any` is a pretty big regression. I'd rather not update the library if that's the primary change.

Reply:
If there's an expectation that the newer version of the library is better then I'm on board

Reply:
I think this is blocked on https://github.com/octokit/rest.js/pull/732